dask_sql/physical/utils/sort.py

```diff
@@ -25,7 +25,7 @@ def apply_sort(
             by=sort_columns,
             ascending=sort_ascending,
             na_position="first" if sort_null_first[0] else "last",
-        )
+        ).persist()
```

Comment:
Noting that I added this persist call after sorting to match up with the other sort paths, which persist after sorting.